bridge/Dockerfile

```
@@ -49,7 +49,7 @@ RUN touch */src/lib.rs && touch */src/main.rs
 RUN cargo build --release --frozen
 
 # Production
-FROM debian:12.1-slim AS prod
+FROM debian:bullseye-slim AS prod
```

```suggestion
FROM debian:11.7-slim AS prod
```
1. I think also being generic is maybe not too awesome, as we can't know what's actually built because of docker cache. I think being explicit is better.
2. Also, numbers are clearer than debian names.

---

Except in this case we are already using debian names in the Rust image, so making the debian image match that IMO *is* clearer.

---

Fair. It doesn't solve the issue about not knowing which version, but that's god enough.